blocks/story-carousel-block/features/story-carousel/default.jsx

```diff
@@ -0,0 +1,138 @@
+
+	// const buildCarousel = () => {
+	//
+	// }
+	return (
+		<div className={BLOCK_CLASS_NAME}>
+			{headline && headline.trim() !== "" && <h1>{headline}</h1>}
+
+			{elements && elements.length && (
```

Comment:
Need to use a ternary otherwise you'll see a `0` output in some cases

```suggestion
			{elements && elements.length ? (
```